elastic/kibana
# [serverless] Add Advanced Settings page

## Summary

This PR adds a Settings application component for rendering the Advanced Settings page in serverless.

### How to test:
1. Start Es with `yarn es serverless` and Kibana with `yarn serverless-{es/oblt/security}`
2. Go to Management -> Advanced Settings
3. Verify that the settings can be changed and saved.

### Advanced Settings page:
<img width="1495" alt="Screenshot 2023-09-28 at 20 56 25" src="https://github.com/elastic/kibana/assets/59341489/374b3bbd-7bf6-4de7-8129-8b293dd1698e">

### Added an Advanced Settings card to the Management landing page:
<img width="1575" alt="Screenshot 2023-09-28 at 12 24 23" src="https://github.com/elastic/kibana/assets/59341489/c08b8b36-ff40-4772-87d6-597629d78342">
